src/Models/Constants/StyleConstants.ts

```diff
@@ -3,7 +3,7 @@ export const behaviorsModalPreviewContainerLeftOffset =
     builderLeftPanelWidth + 36;
 
 export const VIEWER_HEADER_TOP_OFFSET = 10;
-export const BUILDER_HEADER_TOP_OFFSET = 10;
+export const BUILDER_HEADER_TOP_OFFSET = 26;
```

Comment:
why did the vertical spacing change for this?

Reply:
Needed 16 more px of padding, without this, the top controls were misaligned

![image](https://user-images.githubusercontent.com/18181049/172727539-26600703-85cd-45b5-bf1b-c0cf9a7204c2.png)

Reply:
but why? what changed vertically that threw this off? They used to be aligned just fine.

Reply:
now this spacing isn't going to match between builder & viewer mode right? so the controls are going to bounce around.

Reply:
![bouncing](https://user-images.githubusercontent.com/57726991/172734057-52628657-0e74-459a-bf48-cce1216da951.gif)